### ISSUE TITLE
Update the .gitignore and .gitattributes

### ISSUE THREAD

**Issue by GitAdam** [callumW/HauntedHuntingJam#5]
Unreal Engine has a build in Git generator, the missing some ignores should be added to our .gitignore. This project might include some large files, so add git-lfs support.

**Comment by callumW** [comment]
Sounds good! Yeah I just used the default one that github gives in its drop down so it might be a little out of date

**Comment by callumW** [comment]
I had a quick look and it looks like our gitignore has everything the unreal made gitignore has (plus a few extra) so I think we're good there on the gitignore (although I could add *.DS_Store into it, which are macOS files).

We should get git-lfs in asap though.

There are instructions for migrating to git-lfs (i.e. getting already committed files into git-lfs) on the [git-lfs website](https://git-lfs.github.com).

I found a little post on how someone else has been using git-lfs in their project [here](https://awesomeopensource.com/project/MOZGIII/ue4-gitignore).

**But basically**:
When you import a textured mesh into unreal it saves them as *.uassets and there are also *.umap files. So we need to track those files.

And it might be a good idea to also have it track a whole folder for 'raw content' i.e. blender files, sound files, texture images etc. Then we don't have to worry about 'oh I've added a .png, I better add png's into git-lfs' instead, you just add your file in the raw content bin and ```git add``` as normal.

Or we could have a separate repo for raw content... but tbh, that seems like more work and no gain?

Thoughts? @GitAdam
